10_files_lab/fileslab.txt

```diff
@@ -8,30 +8,49 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - .highscore
 
 Write here the commands you used:
+	sudo mkdir lab
```

Comment:
I don't think you need root here.. ```sudo``` is a dangerous command and should be used only when needed.

as a general rule you should never use ```sudo``` when copying files in your home directory, and you should only use it when you get a permission denied error, and you understand the command that you're running :)